04-vector-raster-integration-advanced.Rmd

```diff
@@ -408,31 +437,32 @@ surrounding individual point locations using the `extract()` function. To do thi
 we define the summary method (`fun=mean`) and the buffer distance (`buffer=20`)
 which represents the radius of a circular region around each point.
 
-The units of the buffer are the same units of the data `CRS` - in this case
-`UTM - meters`.
-
+The units of the buffer are the same units of the data `CRS`.
 
 <figure>
     <a href="http://neondataskills.org/images/spatialData/BufferCircular.png">
     <img src="http://neondataskills.org/images/spatialData/BufferCircular.png"></a>
-    <figcaption> Extract raster information using a buffer region. 
+    <figcaption> Extract raster information using a buffer region. All pixels
+    that are touched by the buffer region are included in the extract. 
     Source: National Ecological Observatory Network (NEON).  
     </figcaption>
 </figure>
 
-```{r extract-point-to-buffer }
+Let's put this into practice by figuring out the average tree height in the 
+20m around the tower location. 
 
+```{r extract-point-to-buffer }
 #what are the units of our buffer
 crs(point_HARV)
 
 #extract the average tree height (calculated using the raster pixels)
-#located within the AOI polygon
+#at the tower location
```

Comment:
I like the original way you wrote in the 2-line annotation:
line 1 is saying generally how the process occurs. 
line 2 specifies the parameters we are using.   

What about: 
#extract the average tree height (height is given by the raster pixel value)
#use a buffer of 20 meters and mean function (fun)